src/popup.html

```diff
@@ -17,6 +17,24 @@ <h6 class="card-title">General Settings</h6>
         <input class="form-check-input" type="checkbox" id="renderMap">
         <label class="form-check-label" for="renderMap">Render MapML resources</label>
       </div>
+      <div>
+        <label class="form-label" for="setExtCS">Coordinate system for extent: </label>
+        <select class="form-select" id="defaultExtCoor">
+          <option value="pcrs" selected>pcrs (Default)</option>
+          <option value="gcrs">gcrs</option>
+          <option value="tcrs">tcrs</option>
+          <option value="tilematrix">tilematrix</option>
+        </select>
+        <label class="form-label" for="setLocCS">Coordinate system for location: </label>
+        <select class="form-select" id="defaultLocCoor">
```

Comment:
Change label's "for" attribute so screenreader reads the label, and a small UI suggestion to make the select element smaller.
```suggestion
        <label class="form-label" for="defaultExtCoor">Coordinate system for extent: </label>
        <select class="form-select form-select-sm" id="defaultExtCoor">
          <option value="pcrs" selected>pcrs (Default)</option>
          <option value="gcrs">gcrs</option>
          <option value="tcrs">tcrs</option>
          <option value="tilematrix">tilematrix</option>
        </select>
        <label class="form-label" for="defaultLocCoor">Coordinate system for location: </label>
        <select class="form-select form-select-sm" id="defaultLocCoor">
```